pkgs/tools/misc/overturemaps/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pkgs,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "overturemaps";
+  version = "0.9.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "OvertureMaps";
+    repo = "overturemaps-py";
+    rev = "55a15da30ef8c63f48770a822bbbeab58b124f33";
+    sha256 = "sha256-X3LBibmvh4dGrV4AVmiOZI4Vh+9/2WehA2DwNf+semk=";
+  };
```

Comment:
any reason we don't just fetch from pypi instead? Since there's no tagged releases on GH.

Reply:
I read some advice somewhere to prefer the option that allows for the possibility of running tests, often included in a source repo and not in packaged releases. Happy to change this though.